notifications_utils/statsd_decorators.py

```diff
@@ -13,8 +13,10 @@ def wrapper(*args, **kwargs):
             try:
                 res = func(*args, **kwargs)
                 elapsed_time = monotonic() - start_time
-                current_app.statsd_client.incr("{namespace}.{func}".format(namespace=namespace, func=func.__name__))
-                current_app.statsd_client.timing(
+                current_app.statsd_client.incr(  # type: ignore
```

Comment:
mypy was complaining about 'statsd_client', despite all the relevant tests passing:
`notifications_utils/statsd_decorators.py:16: error: "Flask" has no attribute "statsd_client"`

If there is a better way to solve this rather than ignoring it, please let me know.  (To note: there are several other places where statsd code was likewise ignored like this)